src/Components/Admin/NewUserDialog.js

```diff
@@ -43,6 +43,7 @@ function NewUserDialog({ onSubmit, close }) {
       const user = await createUserAPI(data);
       close();
       onSubmit && onSubmit(user);
+      window.location.reload();
```

Comment:
Let's not use this. 

A state update would just cause the component tree to update and not the whole page.